FEM/pcs_dm.cpp

```diff
@@ -772,19 +772,21 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				if (Norm < Tolerance_global_Newton && Error > Norm)
```

Comment:
It depends on which iterative methods you mean. For Newton iterations, I'm checking only residuals, i.e. norm(r)/norm(r0)<tol. For coupling iterations, displacement changes is checked.